File: C++/1019.cpp
#include <iostream>
using namespace std;

int main()
{
    int sec;
    cin >> sec;
    int h = sec / 3600;
    sec = sec % 3600;
    int m = sec / 60;
    sec = sec % 60;
    int s = sec;
    cout << h << ":" << m << ":" << s << endl;
    return 0;
}

File: PHP/1019.php
<?php
$sec = rtrim(fgets(STDIN));
$hours = floor($sec / 3600);
$sec = fmod($sec, 3600);
$minutes = floor($sec / 60);
$sec = fmod($sec, 60);
echo $hours . ":" . $minutes . ":" . $sec . "\n";
